### PR TITLE
Update teams-calling-policy.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8127
Added note

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-calling-policy.md [hunk 1]
```diff
@@ -54,6 +54,9 @@ Follow these steps to edit an existing calling policy.
 
 Here are the settings that you can configure for calling policies.
 
+> [!NOTE]
+> SIP devices can be used for calls.
```
Comment (JohanFreelancer9) [inline Teams/teams-calling-policy.md:58]:
```suggestion
> You can use SIP devices can be used for calls.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lindspea, kindly commit the suggested change to change the sentence from passive to active voice. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by haneef95** [comment]
Thanks.

would be beneficial to see a link to some documentation on how the SIP devices could be configured.

**Comment by lindspea** [comment]
@haneef95 is this link helpful at all? - https://docs.microsoft.com/microsoftteams/sip-gateway-configure

**Comment by haneef95** [comment]
aha I c, nice thank you @lindspea

**Pull request closed without merging**